examples/jf-control-app/commands/pairing/OpenJointCommissioningWindowCommand.cpp

```diff
@@ -0,0 +1,49 @@
+/*
+ *   Copyright (c) 2025 Project CHIP Authors
+ *   All rights reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ *
+ */
+
+#include "OpenJointCommissioningWindowCommand.h"
+
+#include <system/SystemClock.h>
+
+using namespace ::chip;
+
+CHIP_ERROR OpenJointCommissioningWindowCommand::RunCommand()
```

Comment:
Need to work with @dparausanu / @paulr34 in order to update https://github.com/project-chip/connectedhomeip/blob/master/src/python_testing/TC_JCM_1_2.py with OJCW.